pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.dropwizard.sharding</groupId>
     <artifactId>db-sharding-bundle</artifactId>
-    <version>1.1.4-SNAPSHOT</version>
+    <version>1.1.5-SNAPSHOT</version>
```

Comment:
Change it to 1.1.4-1. As I mentioned it uses 1.1.4 for dropwizard. That's why the name.

Reply:
Ack